packages/cli/internal/signatures.js

```diff
@@ -10,11 +16,30 @@ function getFullFunctionSignature(functionAbi, functionParameters) {
     parameterDescriptions.push(`${input.type} ${input.name}${valueDescription}`);
   }
 
+  // Collect return values
+  const outputDescriptions = [];
+  for (let i = 0; i < functionAbi.outputs.length; i++) {
+    const output = functionAbi.outputs[i];
+
+    outputDescriptions.push(`${output.type}${output.name ? ` ${output.name}` : ''}`);
+  }
```

Comment:
Maybe this is clearer:

```suggestion
  const outputDescriptions = functionAbi.outputs.map((output) =>  {
    return `${output.type}${output.name ? ` ${output.name}` : ''}`
  });
```